### DISCUSSION TITLE
When will any version 5 be promoted to stable edition?

### DISCUSSION THREAD

**Discussion by SHOBU007** [SoftEtherVPN/SoftEtherVPN#2058] [Q&A] [answered]
Hello,

Just a question for anyone that m,ight know the answer.

It's been 2 years since the last release in the softether_stable, and due to my knowledge there's no version 5 that is considered SE yet (am I wrong? I hope I am wrong.)

When will version 5 be promoted to stable edition?

Have a great day and thank you!

**Comment by metalefty** [comment, marked as answer]
No, Stable Edition (ver4) and Developer Edition (ver5) differ in nature and purpose. Contrary to appearances, ver 5 is not a new version (or new version candidate) of ver 4. Both editions are developed under quite different development methodologies. Patches in the Developer Edition might be ported to the Stable Edition however I guess Developer Edition will never become Stable Edition.

See thee @dnobori 's comments for the difference between Developer Edition and Stable Edition:
- https://github.com/SoftEtherVPN/SoftEtherVPN/commit/c2a7aa548137dc80c6aafdc645cf4dc34e0dc764#commitcomment-146254951
- https://github.com/SoftEtherVPN/SoftEtherVPN/commit/6dbf7e9ae27ab3191210742e2075fe08b6c7c43a
